backend/routes/flightStatus.js

```diff
@@ -0,0 +1,46 @@
+import { API_KEY, API_SECRET } from "../config.js";
+import Amadeus from "amadeus";
+import express from "express";
+import { PrismaClient } from '@prisma/client'; 
+import { authenticateUser } from '../auth.js';
+
+const prisma = new PrismaClient();
+const router = express.Router();
+const amadeus = new Amadeus({
+    clientId: API_KEY,
+    clientSecret: API_SECRET,
+});
+
+//flight search for adding trip - needs authentication
+router.get('/flight-status', authenticateUser, async (req, res) => {
```

Comment:
So this "authenticateUser" is a middleware, right?

Reply:
Yeap. I have that function in auth.js and importing it where needed.